gpu documentation updates

georgesvachulay opened the pull request:
# EIDF Documentation Pull Request

## Description

Formatting, structural and content changes introduced with the cluster update of April 2025.

agngrant reviewed (commented):
GPU Service Policies -  Job Names 

- I think focus on the maintenance and troubleshooting - remove the billing at the moment 
-  spec.generateName and spec.name -> is this meant to be metadata.generateName and metadata.name?
-  the important tags - this will make a change in some users job specs, so I think change this to any job not using generateName will be rejected with a reason or similar to that
- the second important tag mentions a lower value - not sure this makes sense here

Kueue:
- the pods are now going to be managed by the Kueue so a kueue name tag will be needed

Tutorial 1 
- can we add the NVIDIA-H200 as an option
Tutorial 2 
- its not been completely decided RBD is going away - so change title of important to Ceph RBD and Ceph FS or similar to highlight the distinction
- maybe put a comment or important marker near the PVC definition to say if you want multiple pods to access the same PVC, this is what ReadWriteMany is
Tutorial 4 
- the section Code development with K8s - the templates and referenced repo are out of date - using name for example

georgesvachulay commented:
Billing removed
metadata fixed
updated important to note changes are required
2nd tag copied by mistake, removed
removed mentions of pods altogether
added h200s
updated pvc to provide options instead of warnings
explanation to RWM added
forked original example repo, updated and changed ref. updated templates in both doc and repo.
